Header Header 
float64 Latitude 
float64 Longitude 
float64 Altitude 
float64 HDOP 
float64 UTM_easting 
float64 UTM_northing 
int64 zone 
string Letter
int32 fix_quality
float64 utc
